c2cgeoportal/views/entry.py

```diff
@@ -584,7 +584,9 @@ def _fill_internal_wms(self, l, layer, wms, wms_layers, errors):
         else:
             if self.default_ogc_server.type != OGCSERVER_TYPE_GEOSERVER:
                 errors.add(
-                    "The layer '{0!s}' is not defined in WMS capabilities".format(wmslayer)
+                    "The layer '{}' ({}) is not defined in WMS capabilities from '{}'".format(
+                        wmslayer, layer.name, self.default_ogc_server.name
```

Comment:
I think that's egal for a human but `layer.layer` isn't more precise ?

And don't we do the same trick (layer.name -> layer.name) for wmts layers ?